l10n_nl_tax_statement/README.rst

```diff
@@ -0,0 +1,197 @@
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--netherlands-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-netherlands/tree/12.0/l10n_nl_tax_statement
+    :alt: OCA/l10n-netherlands
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-netherlands-12-0/l10n-netherlands-12-0-l10n_nl_tax_statement
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/176/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides you with the Tax Statement in the Dutch format.
+
+Use this module in combination with module ``l10n_nl_tax_invoice_basis`` if you want to adopt the *Factuurstelsel* system for your Company.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+* This module depends on module ``date_range``.
+
+Configuration
+=============
+
+This module makes use of the tax tags (eg.: 1a, 1b, 1c, 1d, 2a...) as prescribed by the Dutch tax laws.
+
+If the default Odoo Dutch chart of accounts is installed (module ``l10n_nl``) then these tags are automatically present in the database.
+
+If a non-standard chart of accounts is installed, you have to manually create the tax tags and properly set them into the tax definition.
+The name of the tags must be formatted this way: "+1a (omzet)", "+1a (btw)", "-1a (omzet)", "-1a (btw)", "+2a (omzet)", "+2a (btw)", etc...
+
+If your Company adopts the *Factuurstelsel* system for the accounting, also install the module ``l10n_nl_tax_invoice_basis``
+(for more information about the installation and configuration of that module, check the README file).
+
+The user must belong to the *Show Full Accounting Features* group, to be able to access the `Invoicing -> Configuration -> Accounting -> NL BTW Tags` menu.
+
+Usage
+=====
+
+To create a statement you need to:
+
+#. Verify that you have enough permits. You need to belong to the *Show Full Accounting Features* group.
+#. Go to the menu: `Invoicing -> Reports > Taxes Balance > NL BTW Statement`
+#. Create a statement, providing a name and specifying start date and end date
+#. Press the Update button to calculate the report: the report lines will be displayed in the tab `Statement`
+#. Manually enter the BTW amounts of lines '5d', '5e', '5f' (in Edit mode, click on the amount of the line to be able to change it)
+#. Press the Post button to set the status of the statement to Posted; the statements set to this state cannot be modified
+
+To add past undeclared invoices:
```

Comment:
Maybe it's good to rename to undeclared moves? Since it are not only invoices which can be added later?